doc/src/devdocs/stdio.md

```diff
@@ -2,37 +2,37 @@
 
 ## Libuv wrappers for stdio
 
-`julia.h` defines [libuv](http://docs.libuv.org) wrappers for the `stdio.h` streams:
+julia.h defines [libuv](http://docs.libuv.org) wrappers for the stdio.h streams:
```

Comment:
> anywhere there are wildcards it comes out very wrong without code highlighting

? Line 84 in this file renders fine

> it is an object of sorts

Yes, but not in the same context as the objects that are being referenced around it